package.json

```
@@ -68,7 +76,8 @@
     "ts-loader": "9.2.3",
     "ts-node": "10.9.1",
     "tsconfig-paths": "4.2.0",
-    "typescript": "4.9.5"
+    "typescript": "4.9.5",
+    "husky": "^8.0.0"
```

```suggestion
    "husky": "8.0.0"
```

I would be strict with versioning. 
I would add 
`save-exact=true` to `.npmrc`